SmileIDUI/Classes/SmileIDCaptureScreen.swift

```diff
@@ -13,6 +14,12 @@ public struct SmileIDCaptureScreen<ContinueButton: View>: View {
   ) {
     self.scanType = scanType
     self.onContinue = onContinue
+    self._viewModel = Backport.StateObject(
```

Comment:
we can discuss this in the mobile call, but I was wondering if there is a cleaner way to implement some simple DI that can provide viewmodels across screens